cdap-watchdog/src/test/java/io/cdap/cdap/logging/appender/ErrorClassificationLoggingTest.java

```diff
@@ -54,11 +57,11 @@ public class ErrorClassificationLoggingTest {
   private static Injector injector;
   private static TransactionManager txManager;
 
-  @ClassRule
-  public static final TemporaryFolder TMP_FOLDER = new TemporaryFolder();
+  @Rule
+  public TemporaryFolder TMP_FOLDER = new TemporaryFolder();
```

Comment:
That's pretty expensive to do this per test

Reply:
removed.